### ISSUE TITLE
Ping sample

### ISSUE THREAD

**Issue by stuart-gardner** [newrelic/nri-flex#125]
I am trying to get ping sample working for New Relic on Windows servers. I can do it successfully on my Red Hat servers but on Windows I get nothing. I know Flex is working on Windows as I use it successfully to monitor ports. I would just like to get the continuous ping to work. I have attached the .yml file which I have changed to .txt so I can upload here and this is  located on the Windows server. The ip address listed is for that server.

[ping-example.txt](https://github.com/newrelic/nri-flex/files/4374391/ping-example.txt)

**Comment by ardias** [comment]
Hi @stuart-gardner what version of windows are you using? On my Windows 10 Pro, the ping command should be something like:

`ping -n 5 10.186.6.68`

You also shouldn't need the "|| true" expression.
Also, the output is different between Linux and Windows. In my Windows version I get something similar to :

`Reply from 127.0.0.1: bytes=32 time<1ms TTL=128`
so you will need to account for that in the regex expression you are using to parse the output

**Comment by thezackm** [comment]
created PR #126 to address this. The output of `ping`, `Test-Connection`, and `Test-NetConnection` in  Windows CMD/PowerShell does not emulate the standard output of `ping` in *nix OS. Particularly in the `statistics` block that the sample [ping-url.yml](https://github.com/newrelic/nri-flex/blob/master/examples/flexConfigs/ping-url.yml) targets for output 
ex: 
```
ping www.google.com -c 5
PING www.google.com (172.217.14.228): 56 data bytes
64 bytes from 172.217.14.228: icmp_seq=0 ttl=55 time=62.151 ms
64 bytes from 172.217.14.228: icmp_seq=1 ttl=55 time=63.743 ms
64 bytes from 172.217.14.228: icmp_seq=2 ttl=55 time=60.247 ms
64 bytes from 172.217.14.228: icmp_seq=3 ttl=55 time=59.901 ms
64 bytes from 172.217.14.228: icmp_seq=4 ttl=55 time=62.114 ms

--- www.google.com ping statistics ---
5 packets transmitted, 5 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 59.901/61.631/63.743/1.405 ms
```

The sample in the PR uses a combination of an external PowerShell script and external lookup file to allow the user to execute against any number of targets, each with their own `-Count` parameter.
* windows-ping-example.yml
* windows-ping-script.ps1
* windows-ping-lookup.json

Insights Query: 
```sql
SELECT timestamp, target, packetLoss, packetsReceived, packetsTransmitted, avgResponse, minResponse, maxResponse FROM winPingSample SINCE 1 MINUTE AGO
```

![image](https://user-images.githubusercontent.com/30831476/77453344-241bad80-6dc5-11ea-81a2-586dff04fca9.png)

**_Tested on Server 2019 Datacenter using Flex v1.1.0_**

**Comment by thezackm** [comment]
fixed with #126 , merged to master
